src/services/language-services/hover/layout-hover.ts

```diff
@@ -0,0 +1,142 @@
+import * as vscode from 'vscode';
+
+export const STACK_LAYOUT = vscode.languages.registerHoverProvider('html', {
```

Comment:
Maybe this can be simplified in a way that allows easy adding of new values:
```TypeScript
import * as vscode from 'vscode';

const autocapDescriptions =  { };

const elements = [
    {
        elementName: 'flexboxlayout',
        description: 'A layout based on flexbox.',
        docsUrl: 'https://docs.nativescript.org/ui/layouts/layout-containers',
        apiReferenceUrl: 'https://docs.nativescript.org/api-reference/modules/_ui_layouts_flexbox_layout_'
    },
    {
        elementName: 'wraplayout',
        description: 'A layout that positions children in rows or columns depending on orientation property until space is filled and then wraps them on new row or column.',
        docsUrl: 'https://docs.nativescript.org/ui/layouts/layout-containers',
        apiReferenceUrl: 'https://docs.nativescript.org/api-reference/modules/_ui_layouts_wrap_layout_'
    },
...
];

for (const element of elements) {
    const autocapDescription: vscode.MarkdownString = new vscode.MarkdownString()
        .appendText(`${element.description}\n`)
        .appendText(`\n\n{N}`)
        .appendMarkdown(`  [Docs](${element.docsUrl})`)
        .appendText(`   |   `)
        .appendMarkdown(`[API Reference](${element.docsUrl})`);

    autocapDescription.isTrusted = true;

    autocapDescriptions[element.elementName] = autocapDescription;
}

export const NATIVESCRIPT_HOVER_PROVIDER = vscode.languages.registerHoverProvider('html', {
    provideHover(document, position, token) {
        const range = document.getWordRangeAtPosition(position);
        const word = document.getText(range);
        const autocapDescription = autocapDescriptions[word.toLowerCase()];
        if (autocapDescription) {
            return new vscode.Hover(autocapDescription);
        }
    },
});

export const LAYOUT_HOVER_PROVIDERS = [
    NATIVESCRIPT_HOVER_PROVIDER
];

```